salt/search/gearman-persistence.sls

```diff
@@ -25,28 +25,12 @@ gearman-db:
         - require:
             - postgres_user: gearman-db-user
 
-# 14.04 Upstart only.
+# lsh@2019-09-09: remove once file is gone
 gearman-configuration:
-    file.managed:
+    file.absent:
         - name: /etc/default/gearman-job-server
-        - source: salt://search/config/etc-default-gearman-job-server
-        - template: jinja
-        - require:
-            - gearman-daemon # elife.gearman-server.sls
-            - gearman-db
 
 gearman-service:
-    {% if osrelease == '14.04' %}
-    cmd.run:
-        # I do not trust anymore Upstart to see changes to init scripts when using `restart` alone
```

Comment:
I guess we trust systemd instead? Assuming there's a `systemctl daemon-reload` somewhere

Reply:
I haven't had a problem with daemon-reload + salt for ages now, I suspect it's doing it automatically behind the scenes. I'll keep an eye out for it in future though.